.circleci/config.yml

```diff
@@ -324,89 +195,90 @@ jobs:
             echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
             ' >> ~/.ssh/known_hosts
       - run:
-          name: Notify of new release
+          name: Update Changelog and notify
           command: |
             version=$(get-new-version.sh)
             changelog.sh "$version"
+      - slack/notify:
+          channel: C014UMRC4AJ
+          mentions: here
+          color: '#78c043'
+          message: "🌍 A new Planet 4 release is currently being deployed.\nCheck the <https://support.greenpeace.org/planet4/tech/changelog|Changelog> for the full list of changes"
+          include_job_number_field: false
+          include_project_field: false
+          include_visit_job_action: false
 
 workflows:
-  version: 2
-  branch:
+  develop:
     jobs:
-      - lint
-      - build-branch:
+      - build:
           context: org-global
-          requires:
-            - lint
-          filters:
-            branches:
-              ignore: master
-            tags:
-              ignore: /.*/
-      - test-codeception:
-          context: org-global
-          requires:
-            - build-branch
           filters:
             branches:
-              ignore: master
-      - deploy-develop:
+              only: develop
+      - deploy:
           context: org-global
           requires:
-            - build-branch
+            - build
           filters:
             branches:
               only: develop
-      - notify-promote:
+      - test:
           context: org-global
           requires:
-            - deploy-develop
-            - test-codeception
+            - build
           filters:
             branches:
               only: develop
-      - hold-promote:
+      - hold-trigger:
           type: approval
           requires:
-            - deploy-develop
-            - test-codeception
+            - test
```

Comment:
Maybe it could still be interesting to make this require `deploy` too? `deploy` takes a bit less time than `test`, so it wouldn't slow down the pipeline. It would catch cases where the tests pass, but somehow the code fails to deploy to an instance, which can happen as there's some moving parts to NRO deployment that are not used in the tests (e.g. helm charts).